src/opcodes/flow.cairo

```diff
@@ -14,6 +14,13 @@ pub fn is_branching_opcode(opcode: u8) -> bool {
     return false;
 }
 
+pub fn is_pushing_opcode(opcode: u8) -> bool {
+    if opcode == Opcode::OP_DATA_1 || opcode == Opcode::OP_DATA_75 {
```

Comment:
These aren't all the pushing opcodes, you might want to do a range check too.

Reply:
In the new commit I created the funcion is_data_code to separate the pushing OP_CODES based on their functionality.

- The is_data_opcode for the OP_CODE that just pushes the next bytecode in the stack

- The is_pushing_opcode to the OP_code that the next OP_CODE contains the quantity of bytes